### PR TITLE
Updated with JavaScript Mastery channel 

### PR DESCRIPTION
**Pull request by LimitlessUR**
Hello! I added a channel that has helped me come a long way in JS. I am also new to collaboration. Please let me know if I missed something. Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,4 @@ If you're a YouTuber reading this and if you think you make beginner-friendly tu
 | [Procoder](https://www.youtube.com/c/ProcoderBD) | Makes content around **Web Development** | Bengali |
 | [JS Bangladesh](https://www.youtube.com/c/JSBangladesh) | Makes content around **Web Development** but focuses mostly on **Javascript** | Bengali |
 | [Anisul Islam](https://www.youtube.com/c/anisulislamrubel) | Mostly makes content around **Web Development** but also makes discussion videos & content around other programming languages from time to time | Bengali |
+| [Javascript Mastery](https://www.youtube.com/c/JavaScriptMastery) | Produces great content related to **Javascript Web Development and Application Development** | English |
```
Comment (KendallDoesCoding) [inline README.md:40]:
```suggestion
| [Javascript Mastery](https://www.youtube.com/c/JavaScriptMastery) | Produces great content related to **Javascript Web Development** and **Application Development** | English |
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
the "and" doesn't have to be bold.

**Comment by LimitlessUR** [comment]
Great catch sounds great. Thank you

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @LimitlessUR!, merging.

**Comment by LimitlessUR** [comment]
Thank you :)

**Pull request merged**